src/index.ts

```diff
@@ -161,38 +154,20 @@ export class AzureDataStore {
     });
   }
 
-  /**
-   * Read content from azure blob storage.
-   * @param key
-   * @param callback
-   */
-//   public get (key: any, callback: any): void {
-//     this.opts.blob.getBlobToText(this.container, this.getFullKey(key), { disableContentMD5Validation: true}, (err, text, result, response) => {
-//       if (err && response.statusCode === 404)
-//       {
-//         return callback(Errors.notFoundError(err));
-//       }
-//       else if (response.statusCode !== 200 && err)
-//       {
-//         return callback(err);
-//       }
-//       return callback(null, Buffer.from(text, 'utf-8'));
-//     });
-// }
-
   /**
    * Check for the existence of the given key.
    * @param key
    * @param callback
    */
   public has (key: any, callback: any): void {
+    callback = once(callback);
```

Comment:
Similar to above: I do not understadn, why would callback be called more than once?

Reply:
ipfs s3 implementation had an issue filed for multiple callbacks being called. I checked back now, looks like its not happening, will remove it.